ethereum/datastructures/src/main/java/tech/pegasys/teku/datastructures/util/BeaconStateUtil.java

```diff
@@ -275,6 +275,22 @@ public static UInt64 get_total_balance(BeaconState state, Collection<Integer> in
     return sum.max(EFFECTIVE_BALANCE_INCREMENT);
   }
 
+  /**
+   * Return the combined effective balance of the active validators.
+   *
+   * @param state - Current BeaconState
+   * @return
+   * @see
+   *     <a>https://github.com/ethereum/eth2.0-specs/blob/v0.8.0/specs/core/0_beacon-chain.md#get_total_active_balance</a>
+   */
+  public static UInt64 get_total_active_balance(BeaconState state) {
```

Comment:
Note:  We were [originally caching the squareroot of the total + the total](https://github.com/ConsenSys/teku/commit/55bbed1703001c6d3d412ac37c0d3853f7346f7c#diff-b4374db262b5a5a6020174ebebcad32bf0990864b70803f35614e5d2e0c8990f), simplified by just caching the plain total.